source/includes/tpf/yarrrml.md

```diff
@@ -0,0 +1,332 @@
+    - ['http://purl.org/vocab/bio/0.1/position', 'Emperor', en~lang]
+    - ['http://dbpedia.org/ontology/deathDate', $(death), 'http://www.w3.org/2001/XMLSchema#datetime']
+    - ['http://dbpedia.org/ontology/birthDate', $(birth), 'http://www.w3.org/2001/XMLSchema#datetime']
+    - ['http://dbpedia.org/ontology/birthPlace', city]
+    - ['http://dbpedia.org/ontology/birthPlace', province]
+  city:
+    subject: https://data.opendatasoft.com/ld/resources/roman-emperors@public/City/$(birth_cty)
+    predicateobjects:
+    - [a, 'http://schema.org/City']
+    - [a, 'http://schema.org/Place']
+    - [a, 'http://rdfs.co/juso/SpatialThing']
+    - ['http://www.w3.org/2000/01/rdf-schema#label', $(birth_cty)]
+  province:
+    subject: https://data.opendatasoft.com/ld/resources/roman-emperors@public/Province/$(birth_prv)
+    predicateobjects:
+    - [a, 'http://dbpedia.org/ontology/Province']
+    - [a, 'http://schema.org/Place']
+    - [a, 'http://rdfs.co/juso/SpatialThing']
+    - ['http://www.w3.org/2000/01/rdf-schema#label', $(birth_prv)]
+```
+
+> An example of a record generated in RDF using the previous mapping
+
+```ttl
+<https://data.opendatasoft.com/ld/resources/roman-emperors@public/Person/Augustus> a <http://schema.org/Person>,
+        <http://xmlns.com/foaf/0.1/Person> ;
+    rdfs:label "Augustus" ;
+    ns1:birthDate "0062-09-23T00:53:28+00:53"^^xsd:datetime ;
+    ns1:birthPlace <https://data.opendatasoft.com/ld/resources/roman-emperors@public/City/Rome>,
+        <https://data.opendatasoft.com/ld/resources/roman-emperors@public/Province/Italia> ;
+    ns1:deathDate "0014-08-19T00:53:28+00:53"^^xsd:datetime ;
+    ns2:position "Emperor"@en .
+
+<https://data.opendatasoft.com/ld/resources/roman-emperors@public/City/Rome> a <http://rdfs.co/juso/SpatialThing>,
+        <http://schema.org/City>,
+        <http://schema.org/Place> ;
+    rdfs:label "Rome" .
+
+<https://data.opendatasoft.com/ld/resources/roman-emperors@public/Province/Italia> a ns1:Province,
+        <http://rdfs.co/juso/SpatialThing>,
+        <http://schema.org/Place> ;
+    rdfs:label "Italia" .
+```
+
+This section shows a complete example of an RML mapping for [roman-emperors](https://data.opendatasoft.com/explore/dataset/roman-emperors%40public/table) dataset.
+It also provides more informations about YARRRML and useful tools that facilitate the creation of a RML mapping.
+
+### keys shortcuts
+
+YARRRML allows multiple shortcut versions of keys to be used:
+
+* subjects: subject, s
+* predicateobjects: po
+
+### Tools
+
+Creating a RML mapping for a dataset is not an easy task. However, multiple tools can be really helpful in that process.
+
+[The Linked Open Vocabulary](https://lov.linkeddata.es/dataset/lov/) is a great initiative. It proposes a search engine for vocabularies, properties and classes.
+
+[OpenDataSoft Chatbot (beta)](https://chatbot.opendatasoft.com) is a chatbot that helps user generating a RML mapping for an OpenDataSoft dataset.
```

Comment:
"(...) a chatbot that helps **users generate an** RML mapping for an OpenDataSoft dataset."